frontend/projects/abtesting/src/app/core/http-interceptors/http-error.interceptor.ts

```diff
@@ -4,25 +4,49 @@ import {
   HttpInterceptor,
   HttpHandler,
   HttpRequest} from '@angular/common/http';
-import { Observable, throwError } from 'rxjs';
+import { Observable, EMPTY } from 'rxjs';
 import { catchError } from 'rxjs/operators';
 import { AuthService } from '../auth/auth.service';
+import { environment } from '../../../environments/environment';
+import { NotificationService } from '../notifications/notification.service';
 
 @Injectable()
 export class HttpErrorInterceptor implements HttpInterceptor {
   constructor(
-    private authService: AuthService
+    private authService: AuthService,
+    private notificationsService: NotificationService
   ) { }
 
+  timeOut = 1500;
+  networkErrorCount: number = 0;
```

Comment:
Where are you decrementing the networkErrorCount?

Reply:
You are right, I am working on this.